pinot-core/src/main/java/com/linkedin/pinot/core/io/writer/impl/FixedByteSingleValueMultiColWriter.java

```diff
@@ -24,14 +24,10 @@
 
 public class FixedByteSingleValueMultiColWriter {
   private int[] columnOffsets;
-  private int rows;
   private PinotDataBuffer indexDataBuffer;
   private int rowSizeInBytes;
 
-  public FixedByteSingleValueMultiColWriter(File file, int rows, int cols,
-      int[] columnSizes)
-      throws IOException {
-    this.rows = rows;
+  public FixedByteSingleValueMultiColWriter(File file, int rows, int cols, int[] columnSizes) throws IOException {
```

Comment:
You may want to remove `int rows` here as well.

Reply:
That cannot be removed, because it is used to compute the total size for mmaping in this constructor.